test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -0,0 +1,237 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@graph": [
+    {
+      "@id": "_:000000",
+      "@type": "https://openminds.ebrains.eu/core/SubjectState",
+      "age": {
```

Comment:
The participants.tsv does not contain any age information, so "age" should not be present in the JSON-LD file (i.e. age=None)

Reply:
I manually resolved this in the reference collection but this seems to a problem in bids2openminds. So I created one issue for it too

Reply:
I just remembered, I have solved this issue in a prior pull request #33.